open-feature/open-feature-operator
# fix: Add --port argument in flagd pods for set flagd service port

## This PR


- Enhance argument injecting logic  for set flagd service port (`--port` with FeatureFlagSoruce configuration)

### Related Issues


Fixes #709 

### Notes


### Follow-up Tasks

I don't have the background knowledge of why this feature wasn't implemented, and I haven't tested this code, so a thorough PR review is needed

### How to test

Set `FeatureFlagSource.spec.port` with specific number and check the `--port` argument was properly set and check its debug log at `info    flag-evaluation/connect_service.go:227    Flag IResolver listening at: {PORT_NUMBER}`
